HCCGo/main.js

```diff
@@ -29,5 +29,6 @@ app.on('ready', function() {
 
     mainWindow.once('ready-to-show', () => {
         mainWindow.show();
+        //mainWindow.webContents.openDevTools();
```

Comment:
Potentially, it's more I don't want to make those tools something a normal user will have easy access to. If I'm building a debug version application it's something specifically one of us will be using to test an issue found just before version release. Normal users don't really need it, so would rather not make it too easily accessible.